test/internal/kubernetes/utils_create.go

```diff
@@ -37,62 +40,12 @@ const (
 	envInstallOverlay          = "INSTALL_OVERLAY"
 	envInstallAzureCNIOverlay  = "INSTALL_AZURE_CNI_OVERLAY"
 	envInstallDualStackOverlay = "INSTALL_DUALSTACK_OVERLAY"
```

Comment:
Should we move this to the TestConfig struct?

Reply:
These variables are shared across packages, so I think they should stay here

The TestConfig is only in load